scripts.d/50-vvenc.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+SCRIPT_REPO="https://github.com/fraunhoferhhi/vvenc"
+
+ffbuild_enabled() {
+    [[ $ADDINS_STR == *4.4* ]] && return -1
+    [[ $ADDINS_STR == *5.0* ]] && return -1
+    [[ $ADDINS_STR == *5.1* ]] && return -1
+    [[ $ADDINS_STR == *6.0* ]] && return -1
+    [[ $ADDINS_STR == *7.0* ]] && return -1
+    return 0
+}
+
+fixarm64=()
+
+ffbuild_dockerbuild() {
+
+    mkdir build && cd build
+
+    if [[ $TARGET == *arm64 ]]; then
+    fixarm64=(
+        -DVVENC_ENABLE_X86_SIMD=OFF
+        -DVVENC_ENABLE_ARM_SIMD=OFF
+    )
+    fi
+
+    cmake -DCMAKE_TOOLCHAIN_FILE="$FFBUILD_CMAKE_TOOLCHAIN" -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX="$FFBUILD_PREFIX" "${fixarm64[@]}" ..
+    make -j$(nproc)
+    make install
+}
+
+ffbuild_configure() {
+    echo --enable-libvvenc
+}
+
+ffbuild_unconfigure() {
+    echo --disable-libvvenc
```

Comment:
this will likely also need the same guards as the enable function, since older versions will throw an error due to the unknown option.

Reply:
Thanks, I didn't know that old versions can throw error if libvvenc is not found on FFmpeg old versions. So I removed the line.

Reply:
Hi @BtbN, I brought back with unconfigure command on libvvenc external feature, for older versions will return zero.

Thanks! :)

- Martin Eesmaa

Reply:
Also configuration command too.